components/breadcrumb/navigation.md

```diff
@@ -0,0 +1,40 @@
+---
+title: Navigation
+page_title: Breadcrumb - Navigation
+description: Using the Blazor Breadcrumb for navigating between pages.
+slug: breadcrumb-navigation
+tags: telerik,blazor,breadcrumb,navigation
+published: True
+position: 7
+---
+
+# Breadcrumb for Navigation
+
+The Breadcrumb can be used to navigate between different pages in the application. It can generate the needed links for you through its `UrlField` when [data binding]({%slug breadcrumb-data-binding%}).
+
+In some cases you might prefer to dynamically generate the breadcrumbs based on the current application `Url`. For this scenario you can use the [`NavigationManager`](https://docs.microsoft.com/en-us/dotnet/api/microsoft.aspnetcore.components.navigationmanager?view=aspnetcore-5.0) that the component supports and subscribe to its [`LocationChanged`](https://docs.microsoft.com/en-us/dotnet/api/microsoft.aspnetcore.components.navigationmanager.locationchanged?view=aspnetcore-5.0) event. Thus, you can set new Breadcrumb items on every location change.
+
+>caption To use the Breadcrumb and dynamically generate its items:
+
+* Add the Breadcrumb to your application.
+    * You may want to add it in the `MainLayout.razor` outside of the `@Body`, for example, in the `main` container of your app.
+
+* Introduce a `NavigationManager` instance by injecting it in the file you would like to build the breadcrumbs.
```

Comment:
There is no need to suggest and action as a concept and then explain how to do it. Directly tell what to do.

```suggestion
* Inject a `NavigationManager` instance in the file where you want to build the breadcrumbs.
```